### ISSUE TITLE
Schema not specified for `COMMENT ON INDEX` statements, leading to RELATION ... DOES NOT EXIST

### ISSUE THREAD

**Issue by olivierdalang** [michaelsogos/pg-diff#44]
Hi !

In my scripts, the following statements are generated :
```sql
COMMENT ON INDEX sidx_landvaluezone_geom  IS NULL;
```

They fail to apply with error `ERROR: relation "sidx_landvaluezone_geom" does not exist`.

If I manually specify the schema 
```sql
COMMENT ON INDEX topo.sidx_landvaluezone_geom  IS NULL;
```
it works as expected.

**Comment by michaelsogos** [comment]
Thanks @olivierdalang 

It will be fixed soon! 😄 

Which version are you using? Which project, API or CLI?

**Comment by olivierdalang** [comment]
Using pg-diff-cli version 2.4.0

Awesome thanks !

**Comment by michaelsogos** [comment]
Dear @olivierdalang 

Sorry for the late.

Fix has been released on CLI v2.4.1 and API v1.4.1.

Do you like to give a try ?
